extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -1304,7 +1412,7 @@ export async function setModelValues(inputComponents: InputComponents, model: Mo
 	}));
 }
 
-async function getInputComponentValue(inputComponentInfo: InputComponentInfo): Promise<InputValueType> {
+async function getInputComponentValue(inputComponentInfo: InputComponentInfo<InputComponent>): Promise<InputValueType> {
```

Comment:
Shouldn't this method be useing the inputComponentInfo.value to get inputComponentInfo.component value instead of the direct component value here ...

Reply:
Yeah good call - I have the changes for that but I'm going to do them in another PR since it's a decent chunk of changes.